integration/spark2/src/main/scala/org/apache/spark/sql/execution/command/management/CarbonLoadDataCommand.scala

```diff
@@ -94,6 +89,8 @@ case class CarbonLoadDataCommand(
 
   var table: CarbonTable = _
 
+  var logicalRelation: LogicalRelation = _
```

Comment:
in some scenarios, the case class command should be serializable.

Reply:
Yes I know that, is LogicalRelation cannot be serialized in those scenarios?

Reply:
@QiangCai I have verified by serializing LogicalRelation , and it is able to serialize and deserialize with out any issue. Only it could not tableSize and that also i added t class level.

Reply:
OK, I also confirmed that.